src/_provisional.scss

```
@@ -158,4 +158,11 @@
 			color: #fff;
 		}
 	}
+
 }
+
+.provisional {
+	//This color will eventually be replaced by the new colors from the alert revamp PR.
+	&.icon-warning-light, .icon-warning-light {
+		color: #DF7200;
+}}
```

Can you please put the curly brackets like this instead of on one line?
```suggestion
.provisional {
	//This color will eventually be replaced by the new colors from the alert revamp PR.
	&.icon-warning-light, .icon-warning-light {
		color: #DF7200;
	}
}
```